app/src/test/java/com/team7/cmput301/android/theirisproject/CareProviderTest.java

```diff
@@ -33,8 +33,10 @@ public void testAddPatient() {
         CareProvider careProvider = getTestCareProvider();
         Patient patient = getTestPatient();
 
+        Assert.assertEquals(careProvider.getPatients().size(), 0);
         careProvider.addPatient(patient);
         Assert.assertEquals(careProvider.getPatients().size(), 1);
+        Assert.assertEquals(careProvider.getPatients(), patient);
```

Comment:
This should probably get careProvider.getPatients().get(0), otherwise you're comparing the PatientsList with the test patient

Reply:
Make sure to fix this in the other places like RecordPhoto